src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -784,27 +825,40 @@ public override bool CanFinishEditing(IEnumerable<EditorUserOverride> userOverri
         return true;
     }
 
+    public void SetPatchConditions(Patch patch)
+    {
+        editorPatchConditions = (BiomeConditions)patch.Biome.Clone();
+
+        // If the editor has initialised (i.e. if this is a change of patch during an editor session), switch back to
+        // daytime light levels
+        if (Editor.Ready)
```

Comment:
I think instead of detecting this, this method should take in a bool that's true when things are being initialized.